src/App.tsx

```diff
@@ -1,10 +1,51 @@
 /* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
+import React, { useEffect, useState } from 'react';
+import { getTodos } from './api/todos';
+import { Footer } from './components/Footer';
+import { Header } from './components/Header';
+import { Notification } from './components/Notification';
+import { Todos } from './components/Todos';
+import { FilterOptions } from './types/FilterOptions';
+import { Todo } from './types/Todo';
 import { UserWarning } from './UserWarning';
 
-const USER_ID = 0;
+const USER_ID = 6133;
 
 export const App: React.FC = () => {
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [filter, setFilter] = useState('all');
+  const [notification, setNotification] = useState('');
+  const [hideNotification, setHideNotification] = useState(false);
+
+  const handleHideNotification = () => setHideNotification(true);
+  const handleError = (error: string) => {
+    setNotification(error);
+    setTimeout(() => {
+      setHideNotification(true);
+    }, 3000);
+  };
```

Comment:
it would be better to move it to Notification component and call setTimeout in useEffect with [error] dependency in order to reuse it whenever you want.